Bottom: a08a632ce6615c6d2be270fe605a203c5167da4f
Top:    840a02768ac75223df2878a40be1de19e9b0b568
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 15:54:27 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/lib/master.c b/lib/master.c
index 7d92bb1..269ea10 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1431,8 +1431,6 @@ int master_show_mounts(struct master *master, const char *maps)
 
 		if (!this->maps) {
 			printf("  no map sources found\n\n");
-			if (map_path)
-				free(map_path);
 			continue;
 		}
